### ISSUE TITLE
music icon not working, music play song not working

### ISSUE THREAD

**Issue by chekovma** [xbmc/Kore#391]
Two probably separate issues:
1.  Pressing the headphones icon at the bottom of the main screen when connected to a Kodi 17 media center (I tested with 17.1 and 17.3) does nothing.  All of the other icons (movies, tv shows, etc) work, but the music icon has no effect.
2.  When navigating to the music section of the remote (via the kore menu) I am able to see artists, albums, etc.  Clicking on an album and pressing play properly starts playing at track 1 and queues the rest of the album.  Clicking on an Artist then a song or on a song from the Songs list and selecting "add to playlist" returns a "Couldn't queue media file" error siting "Invalid params..."  Clicking on Play causes the media center to start "Party Mode" ignoring the selected song....
I'm running Kore 2.2.0 on Android 7.1.2

**Comment by tomerf** [comment]
I tested the latest version and it is working for me.
1. I fixed it but it's still not in the Play store, can you check with the latest build?
2. I didn't had any problem, can you explain how to reproduce it?

**Comment by chekovma** [comment]
1. ok I'll test it out
2.  I noticed someone in the forums had mentioned it previously:
http://forum.kodi.tv/showthread.php?tid=308520
So perhaps it has already been fixed in the latest build.  I forgot to mention the "invalid params" probem when adding to a playlist as well.  To reproduce:
Basically Kore->Menu->Music->Songs then pick a song and press play.  Kodi will, instead of playing that song, start a party mode playlist.

thanks
